src/data/api.js

```diff
@@ -4,15 +4,52 @@ import { ExamAction } from '../constants';
 
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
+async function fetchActiveAttempt() {
+  const activeAttemptUrl = new URL(`${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`);
+  const activeAttemptResponse = await getAuthenticatedHttpClient().get(activeAttemptUrl.href);
+  return activeAttemptResponse.data.attempt;
+}
+
 export async function fetchExamAttemptsData(courseId, sequenceId) {
-  const url = new URL(
-    `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
-  );
-  if (sequenceId) {
-    url.searchParams.append('content_id', sequenceId);
+  let data;
+  if (!getConfig().EXAMS_BASE_URL) {
+    const url = new URL(
+      `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
+    );
+    if (sequenceId) {
```

Comment:
this should always be called with a sequence id now right?

Reply:
Let me verify, but I'm pretty sure

Reply:
Yeah, it should always have a sequence ID now. Will remove